src/test/resources/specs/name-matching/Last_names_combination.feature

```diff
@@ -120,7 +120,7 @@ Feature: Last name combinations
       | nino          | NR123456C     |
     Then the following names will be tried
       | First name    | Last name |
-      | Arthur Brian  | O Coates  |
+      | Arthur        | O Coates  |
```

Comment:
It's only going to affect the edge case where the very first name is shorter than 3 characters but this change is maybe not one we should be making.

The Special Characters generator owns a Multiple Last Names strategy which used to add the entire name. I think perhaps it should now own a EntireNonAliasName generator so that we still get "Arthur Brian O Coates"?